sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -817,6 +818,18 @@ case class Sample(
     child: LogicalPlan)(
     val isTableSample: java.lang.Boolean = false) extends UnaryNode {
 
+  val eps = RandomSampler.roundingEpsilon
```

Comment:
OK. I'm not sure the same issue applies here because the bounds aren't being generated programmatically. This would just be an argument from user code directly right?